src/boot_animation.py

```diff
@@ -42,19 +42,19 @@
     # set sample rate & bit depth
     dac.configure_clocks(sample_rate=11030, bit_depth=16)
 
-    if "sound" in launcher_config:
-        if launcher_config["sound"] == "speaker":
+    if "ltv320" in launcher_config:
```

Comment:
I think the letters in the name of the dac has gotten transposed a little. 

the Fruit Jam dac is `TLV320`

Reply:
Thanks!!!!